earthpy/spatial.py

```diff
@@ -77,15 +77,25 @@ def normalized_diff(b1, b2):
     --------
     >>> import numpy as np
     >>> import earthpy.spatial as es
+<<<<<<< HEAD
```

Comment:
@jlpalomino this PR is not ready for review. you will need to resolve these conflicts before anything will build properly and before someone can review things. Please do that first and look closely at your PR before submitting to ensure these types of things are cleaned up!! thank you!!

Reply:
@lwasser My fork has been updated, and these conflicts have now been resolved.  This PR is ready to merge.